docs/notebooks/interpret_advanced_usage.ipynb

```diff
@@ -0,0 +1,2393 @@
+{
```

Comment:
<p>I think you can do add <code>.item()</code>  to the last line and it will just show the number, without all the DataArray metadata which may be distracting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/762/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Yeah, it's much cleaner with `.item()`